src/TestUtils/src/UITest.NUnit/UITestBase.cs

```diff
@@ -105,7 +111,7 @@ public void OneTimeTearDown()
 			{
 				SaveDeviceDiagnosticInfo();
 
-				if (App.AppState != ApplicationState.NotRunning)
+				//if (App.AppState != ApplicationState.NotRunning)
```

Comment:
Does this line need to be removed?

Reply:
I'm mainly in the process of trying to figure out why `InputTransparencyWhenRootIsTransparentMatrix` keeps failing and seeing if I can tweak the logfiles to give me more info. 

If I get it figured out :-) then I'll clean things up